lib/hanami/cli/application/cli.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
-require "hanami/cli"
+require "dry/cli"
 require_relative "commands"
 
 module Hanami
   class CLI
     module Application
       # Hanami application CLI
-      class CLI < Hanami::CLI
+      class CLI < Dry::CLI
```

Comment:
@timriley Please correct me if I'm wrong, were you going to make 
`Hanami::CLI::Application::CLI.new(Hanami::CLI::Commands)` in the `bin/hanami` ?